src/components/Publication/Author.tsx

```diff
@@ -17,18 +17,23 @@ const NotLabMember = styled.span`
   color: ${Color.gray500};
 `
 
-export const Author = ({ authors }: { authors: (Member | string)[] }) => {
-  return (
-    <AuthorList>
-      {authors.map(author =>
-        typeof author === 'string' ? (
-          <NotLabMember key={author}>{author}</NotLabMember>
-        ) : (
-          <LabMember key={author.email}>
-            {author.firstName} {author.lastName}
-          </LabMember>
-        )
-      )}
-    </AuthorList>
+const printAuthor = (author: Member | string, asterisks: string) => {
```

Comment:
Since this returns a component, the name should be `PrintAuthor`.  I would change the name to `Name`